packages/seed/src/commands/test/testWorkspaceFixtures.ts

```diff
@@ -228,21 +246,55 @@ async function testWithWriteToDisk({
             fixtureName: fixture
         });
         for (const script of scripts ?? []) {
+            taskContext.logger.info(`Running script on ${fixture}`);
+            const workDir = `${fixture}_${outputFolder}`;
             const scriptFile = await tmp.file();
-            await writeFile(scriptFile.path, ["cd /generated", ...script.commands].join("\n"));
-            taskContext.logger.info(`Running script ${scriptFile.path}`);
+            await writeFile(scriptFile.path, [`cd /${workDir}/generated`, ...script.commands].join("\n"));
+
+            // Move scripts and generated files into the container
+            const mkdirCommand = await loggingExeca(
+                taskContext.logger,
+                "docker",
+                ["exec", script.containerId, "mkdir", `/${workDir}`],
+                {
+                    doNotPipeOutput: true
+                }
+            );
+            const copyScriptCommand = await loggingExeca(
+                undefined,
+                "docker",
+                ["cp", scriptFile.path, `${script.containerId}:/${workDir}/test.sh`],
+                {
+                    doNotPipeOutput: true
+                }
+            );
+            const copyCommand = await loggingExeca(
+                taskContext.logger,
+                "docker",
+                ["cp", `${outputDir}/.`, `${script.containerId}:/${workDir}/generated/`],
+                {
+                    doNotPipeOutput: true
+                }
+            );
+            if (copyScriptCommand.failed || mkdirCommand.failed || copyCommand.failed) {
+                taskContext.logger.error("Failed to run copy of scripts. See ouptut below");
+                taskContext.logger.error(copyCommand.stdout);
+                taskContext.logger.error(copyCommand.stderr);
+                return { type: "failure", reason: "Failed to run script...", fixture };
```

Comment:
@armandobelardo small bug here that we are printing just the copyCommand.stdout but checking failure for all of the 3 commands; i think it might be better to have 3 separate if statements for each command in case it fails

Reply:
That's fair, will do! I was originally trying to keep it a bit more concise, but that doesn't make much sense haha